common/src/commonMain/kotlin/org/hisp/dhis/common/screens/ImageBlockScreen.kt

```diff
@@ -0,0 +1,13 @@
+package org.hisp.dhis.common.screens
+
+import androidx.compose.runtime.Composable
+import org.hisp.dhis.mobile.ui.designsystem.component.ImageBlock
+import java.io.File
+
+@Composable
+fun ImageBlockScreen() {
+    ImageBlock(
+        file = File("/data/data/org.hisp.dhis.android/files/sample.jpg"),
```

Comment:
maybe you can add a sample image jpg in the `commons` module and then use it here to test the component

Reply:
Instead of using a packaged resource. We can add an option to pick a file from the device storage in the catalog app? Since that's will be actual use case no?

Reply:
ok, but this would made us handling permissions right?. I mean mean while is in commons and not in desigsystem module both solutions will look good for me

Reply:
Yeah, but we would be handling packaged resource and this block expects a file path. I don't think that would work, no? The other approach we were discussing is to create a `Painter` and we can pass any painter like `painterResource` or `painterFile` (we create the painter file). 

With that we can pass `painterResource` in catalog app and use `painterFile` in production if that works.

Reply:
@andresmr @msasikanth I have update the Image Block to make it more generic, now we can pass the type of resource we want to load such as ImageBitmap, Painter, etc. And painter parameter to define how to paint it.
**For Loading Image from Local Storage**
`ImageBlock(
        load = { loadImageBitmap() },
        painterFor = { remember { BitmapPainter(it!!) } },
        onClick = {},
    )`
    
**For Loading Image from Resource**
`ImageBlock(
        load = { painterResource("drawable/sample.png") },
        painterFor = { remember { it } },
        onClick = {},
    )`